packages/httpd.rb

```diff
@@ -3,21 +3,22 @@
 class Httpd < Package
   description 'The Apache HTTP Server Project is an effort to develop and maintain an open-source HTTP server for modern operating systems including UNIX and Windows.'
   homepage 'http://httpd.apache.org/'
-  version '2.2.32'
-  source_url 'https://github.com/apache/httpd/archive/2.2.32.tar.gz'
-  source_sha256 '0d74b1e4d69cedc56323cedc819c083ca975b679b6a082abf60c21836756bb1d'
+  version '2.4.27'
+  source_url 'http://www-us.apache.org/dist/httpd/httpd-2.4.27.tar.gz'
+  source_sha256 '346dd3d016ae5d7101016e68805150bdce9040a8d246c289aa70e68a7cd86b66'
 
-  depends_on 'subversion'
+  depends_on 'apr'
+  depends_on 'aprutil'
+  depends_on 'libtool'
+  depends_on 'pcre'
 
   def self.build
-    system 'svn co http://svn.apache.org/repos/asf/apr/apr/branches/1.4.x srclib/apr'
-    system 'svn co http://svn.apache.org/repos/asf/apr/apr-util/branches/1.4.x srclib/apr-util'
-    system './buildconf'
-    system './configure --prefix=/usr/local'
+    system './configure'
     system 'make'
   end
 
   def self.install
-    system 'make -i install'
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "libtool --finish #{CREW_DEST_DIR}#{CREW_PREFIX}/apache2/modules"
```

Comment:
I just followed the instructions from the message during compilation:
```
libtool: warning: remember to run 'libtool --finish /usr/local/apache2/modules'
```
I'm not sure if the `#{CREW_DEST_DIR}` prefix adversely affects the install or not but it appears to be working fine.